src/platform/pump_events.rs

```diff
@@ -2,7 +2,7 @@ use std::time::Duration;
 
 use crate::{
     event::Event,
-    event_loop::{ControlFlow, EventLoop, EventLoopWindowTarget},
+    event_loop::{EventLoop, EventLoopWindowTarget},
 };
 
 /// The return status for `pump_events`
```

Comment:
`PumpStatus` also no longer need the exit code

Reply:
I was just about to write an issue for that. It's still used by Linux and Windows to return error codes. Will post in a couple.

Reply:
https://github.com/rust-windowing/winit/issues/3059.